### PR TITLE
Some more badges for README

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality.

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by CR
- [x] tests executed by CR, DN

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -253,6 +255,7 @@ be found in [CONTRIBUTING](CONTRIBUTING.md).
 [loqusdb-scout]: docs/admin-guide/loqusdb_integration.md
 [gens]: https://github.com/Clinical-Genomics-Lund/gens
 [gens-scout]: docs/admin-guide/gens_integration.md
+[github-release-date]: https://img.shields.io/github/release-date/Clinical-Genomics/scout
```
Comment (northwestwitch) [inline README.md:258]: 👍🏻

File: README.md [hunk 2]
```diff
@@ -7,12 +7,14 @@
 
 ![Build Status - GitHub][actions-build-status]
 [![PyPI Version][pypi-img]][pypi-url]
-[![Coverage Status][codecov-img]][codecov-url]
+![GitHub Release Date][github-release-date]
+[![GitHub commits latest](https://img.shields.io/github/commits-since/Clinical-Genomics/scout/latest)](https://GitHub.com/Clinical-Genomics/scout/commit/)
+[![GitHub commit rate](https://img.shields.io/github/commit-activity/w/Clinical-Genomics/scout)](https://GitHub.com/Clinical-Genomics/scout/pulse/)
+[![GitHub commits](https://img.shields.io/github/commits-since/Clinical-Genomics/scout/v4.8.0.svg)](https://GitHub.com/Clinical-Genomics/scout/commit/)
```
Comment (northwestwitch) [inline README.md:13]: This is one is weird, why from 4.8? Not your fault, it was already like this!

image

**Review by northwestwitch — CHANGES_REQUESTED** [review]
Perhaps remove the one with version 4.8?

**Review by northwestwitch — APPROVED** [review]
💯

File: README.md [hunk 3]
```diff
@@ -10,7 +10,7 @@
 ![GitHub Release Date][github-release-date]
 [![GitHub commits latest](https://img.shields.io/github/commits-since/Clinical-Genomics/scout/latest)](https://GitHub.com/Clinical-Genomics/scout/commit/)
 [![GitHub commit rate](https://img.shields.io/github/commit-activity/w/Clinical-Genomics/scout)](https://GitHub.com/Clinical-Genomics/scout/pulse/)
-[![GitHub commits](https://img.shields.io/github/commits-since/Clinical-Genomics/scout/v4.8.0.svg)](https://GitHub.com/Clinical-Genomics/scout/commit/)
+[![GitHub commits](https://img.shields.io/github/commits-since/Clinical-Genomics/scout/v4.0.0.svg)](https://GitHub.com/Clinical-Genomics/scout/commit/)
```
[the remark this answers is not in the document]
Comment (dnil) [reply README.md:13]: Alas, our days are finite, but work goes on, so perhaps yes! 😁

File: README.md [hunk 4]
```diff
@@ -17,6 +17,7 @@
 [![Coverage Status][codecov-img]][codecov-url]
 [![CodeFactor](https://www.codefactor.io/repository/github/clinical-genomics/scout/badge)](https://www.codefactor.io/repository/github/clinical-genomics/scout)
 [![Code style: black][black-image]][black-url]
+[![Woke][woke-image]][woke-url]
```
Comment (northwestwitch) [inline README.md:20]: 🥇

File: README.md [hunk 5]
```diff
@@ -7,14 +7,17 @@
 
 ![Build Status - GitHub][actions-build-status]
 [![PyPI Version][pypi-img]][pypi-url]
-[![Coverage Status][codecov-img]][codecov-url]
+![GitHub Release Date][github-release-date]
+[![GitHub commits latest](https://img.shields.io/github/commits-since/Clinical-Genomics/scout/latest)](https://GitHub.com/Clinical-Genomics/scout/commit/)
+[![GitHub commit rate](https://img.shields.io/github/commit-activity/w/Clinical-Genomics/scout)](https://GitHub.com/Clinical-Genomics/scout/pulse/)
+[![GitHub commits](https://img.shields.io/github/commits-since/Clinical-Genomics/scout/v4.0.0.svg)](https://GitHub.com/Clinical-Genomics/scout/commit/)
```
Comment (northwestwitch) [inline README.md:13]: Ah look, this one didn't work: 
image

Comment (northwestwitch) [reply README.md:13]: Ah no, it the other one. Never mind!

Comment (dnil) [reply README.md:13]: Ah, but that is latest; the slightly large one is there as well:
Screenshot 2022-11-23 at 15 43 17

**Pull request merged**